docs-src/content/using-diatheke-sdk/session/event-stream.md

```diff
@@ -480,6 +540,26 @@ client.commandFinished(sessionID: sessionID, commandStatus: status)
 session.commandFinished(commandStatus: status)
 {{< /tab >}}
 
+{{< tab "Java/Android" "java" >}}
+DiathekeOuterClass.CommandStatus status = DiathekeOuterClass.CommandStatus.newBuilder().build();
+mClient.commandFinished(status, new StreamObserver<DiathekeOuterClass.Empty>() {
```

Comment:
Why not use the [blocking unary version](https://grpc.io/docs/reference/java/generated-code/#unary-2) instead of the asynchronous version? That would be more consistent with what the other languages we support do.

Reply:
I add example but i think it is not very important. 
When you use blocking unary  you should use try catch block and it is looks like:
DiathekeGrpc.DiathekeFutureStub diathekeFutureStub = DiathekeGrpc.newFutureStub(mCubicChannel);
        DiathekeOuterClass.CommandStatus status = DiathekeOuterClass.CommandStatus.newBuilder().build();
        ListenableFuture<DiathekeOuterClass.Empty> emptyListenableFuture = diathekeFutureStub.commandFinished(status);
        DiathekeOuterClass.Empty empty;
        try {
             empty = emptyListenableFuture.get();
        } catch (InterruptedException | ExecutionException e) {
            e.printStackTrace();
            empty=null;
        }

If you want i can add alternative example with FutureStub

Reply:
No. I now realize that once the client stub is created, we can only use that version of methods. Since we need the asynchronous stub for some of our other methods, we should stick to the asynchronous version for everything.